gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -653,7 +653,7 @@ private static GoogleCloudStorageReadOptions getReadChannelOptions(Configuration
         .build();
   }
 
-  private static AsyncWriteChannelOptions getWriteChannelOptions(Configuration config) {
+  public static AsyncWriteChannelOptions getWriteChannelOptions(Configuration config) {
```

Comment:
though this method isn't strictly needed based on my Beam PR, I thought it made sense to make it public as well to have symmetry with the `getReadChannelOptions` method.